README.md

```diff
@@ -353,10 +353,10 @@ to applications using
 [`StorageClass
 Parameters`](https://kubernetes.io/docs/concepts/storage/storage-classes/#parameters). An
 optional `persistencyModel` parameter differentiates how the
-provisioned volume can be used.
+provisioned volume can be used:
 
-* if no `persistencyModel` parameter specified in `StorageClass` then
-  it is treated as normal Kubernetes persistent volume. In this case
+* no `persistencyModel` parameter or `persistencyModel: normal` in `StorageClass`  
```

Comment:
extra spaces (2 of them) at the end of line. Note that GH diff view shows me just one (is it so that one gets added by this change?)
 `git diff` on cmdline shows two.
Note also there are few other occurrences in the file of extra space at end of bullet-line (and others), but outside of scope of this change.

Reply:
Two spaces are intentional, please don't remove them: https://meta.stackexchange.com/questions/72395/is-it-possible-to-have-definition-lists-in-markdown/298872#298872

Reply:
Thanks for pointer, I did not know that feature. IMHO it is not so good idea to use trailing spaces for formatting, but if it's so then let's try to maintain those.
Not looking so good via `git diff` which indicates trailing whitespace as problem.